weave-js/src/components/Panel2/pyBoardGen.tsx

```diff
@@ -11,13 +11,44 @@ import {
   absoluteTargetMutation,
   makeMutation,
   useClientContext,
+  useNodeValue,
   useRefreshAllNodes,
 } from '@wandb/weave/react';
-import {useCallback} from 'react';
+import {useCallback, useMemo} from 'react';
 
 import {usePanelContext} from './PanelContext';
 import moment from 'moment';
 
+export const useBoardGeneratorsForNode = (
+  node: Node,
+  allowConfig: boolean = false
+): {
+  loading: boolean;
+  result: Array<{
+    display_name: string;
+    description: string;
+    op_name: string;
+  }>;
+} => {
+  const genBoardsNode = callOpVeryUnsafe(
```

Comment:
I'd like to learn more about `callOpVeryUnsafe` at some point

Reply:
TLDR: as our Python op library outgrows JS, they are no longer defined in JS. This allows us to explicitly stitch a new node in the graph, but is "unsafe" since we cannot be guaranteed such op exists, or that the params are correct, or that the type is correct.